onboarding/sprint-rotation.md

```diff
@@ -30,6 +30,14 @@ Following the sprint rotation, the new engineer will join a team full-time.
   - Schedule of tech leads/PMs
   - Specific areas of interest for new engineer
 - Manager makes sure the tech lead/PM are aware (well in advance) of the upcoming rotation.
+- Manager selects a rotation buddy.
+
+##### Rotation buddy
+
+The goal of the rotation buddy is make the onboardee feel comfortable on their rotation team and provide context and information about ongoing projects. The rotation buddy may be their mentor for one rotation, but the goal is not to provide mentorship. Responsibilities include:
```

Comment:
I think the "the goal is not to provide mentorship" is slightly misleading! It seems clear enough without that caveat.

Reply:
Reading @joeyAghion's description of the rotation buddy, I think that a new section isn't necessary, so I will scrap these changes.